ansible/roles/test/files/tools/loganalyzer/loganalyzer_init.yml

```diff
@@ -8,8 +8,8 @@
   copy: src="{{ loganalyzer_location }}/loganalyzer.py" dest="{{ run_dir }}"
 
 - set_fact:
-    testname_unique: "{{ testname }}.{{lookup('pipe','date +%Y-%m-%d-%H:%M:%S')}}"
-  when: testname_unique is not defined
+    testname_unique: "{{ testname }}.{{ ansible_date_time.date }}.{{ lookup('pipe', 'date +%H:%M:%S') }}"
```

Comment:
If you like to use "." to separate date and time, you can consider below format too:
```
testname_unique: "{{ testname }}.{{ lookup('pipe', 'date +%Y-%m-%d.%H:%M:%S') }}"
```